.github/custom_python_actions/compliance_checks.py

```diff
@@ -38,7 +38,7 @@ def get_team_name(code_owners: str, org_name: str) -> str:
     if len(team_handle) > 1:
         raise Exception("Only one team can be listed for repo-level codeowners.")
     codeowner_team = team_handle[0].lower()
-    team_name = codeowner_team.strip(f"@{org_name.lower()}").strip("/")
+    team_name = codeowner_team.replace(f"@{org_name.lower()}/", "")
```

Comment:
Add test for this?

Reply:
We actually already have tests for this here: https://github.com/dfinity/repositories-open-to-contributions/blob/355ccd7a907290aa24434737ad3714b7ba2294c5/.github/tests/test_compliance_checks.py#L67

It was a very bizarre error - it seemed to only be an issue with the letter "t"? If we want to be thorough, I can add another test for that specific case, but because the tests still passed after this change I feel confident with it.

Reply:
Just ad a case that would fail on the current code and pass with this change?